src/fio/fioEngine.js

```diff
@@ -1440,7 +1445,10 @@ export class FioEngine extends CurrencyEngine {
       this.recentFioFee = { publicAddress, fee }
     }
 
-    if (bns.gt(bns.add(quantity, `${fee}`), nativeBalance)) {
+    if (
```

Comment:
Add a comment explaining why we don't need to check the available balance for unstaked action (because that's handled separately below).